Doc Update for Deploy Status (#1881)
astronomer/docs: astro/deploy-code.md

## Step 3: Validate your changes

> During a deploy in the Cloud UI, the status of the Deployment is **Deploying**. Hover over the status indicator to view specific information about the deploy, including whether it was an image deploy or a DAG-only deploy. After the deploy completes, the **Docker Image** field in the Cloud UI provides Docker image information for the Deployment. Depending on how your organization deploys to Astro, the **Docker Image** field displays a unique identifier generated by a Continuous Integration (CI) tool or a timestamp generated by the Astro CLI on `astro deploy`. The **Docker Image** field identifies the Docker image running on the webserver of your Deployment. This information can help you determine if your deploy was successful, but it does not identify the Docker image running on your scheduler, triggerer, or workers. To confirm a code push with complete certainty, contact [Astronomer support](https://cloud.astronomer.io/support).

Reviewer: ```suggestion
After you initialize a code deploy to Astro, the status of the Deployment is **DEPLOYING** until the underlying Airflow components are determined to be running the latest version of your code. During this time, hover over the status indicator to view specific information about the deploy, including whether your entire Astro project was deployed or only DAGs.

After the deploy completes, the **Docker Image** and **DAG bundle version** fields in the Cloud UI will update with the version of your code that is running on the Deployment. The **Docker Image** field displays a unique identifier generated by a Continuous Integration (CI) tool or a timestamp generated by the Astro CLI on `astro deploy`. The **DAG bundle Version** displays a unique timestamp generated by the Astro CLI when DAGs are updated. If you do not have DAG-only deploys enabled, the DAG bundle version will not appear.

To confirm that your deploy was successful, verify that the running versions are correct.
```

Running out of time to polish this, but lemme know what you think here @kwcanuck

Author: @paolaperaza we shouldn't use _initialize_ as it isn't simple English and the style guide advises against its use:

![image](https://user-images.githubusercontent.com/104205257/218742029-669950cd-d92c-496a-bf8e-e4b6dfb9a7d5.png)

I've updated your proposed text to use the simpler word _start_ and made other changes for brevity, accuracy, and style.

1. In the Cloud UI, select a Workspace and then select a Deployment.

> 2. Review the information in the **Docker Image** field.

Reviewer: Be more specific about what users can expect to review

Author: @paolaperaza I've added the requested information.

## Step 3: Validate your changes

When you start a code deploy to Astro, the status of the Deployment is **DEPLOYING** until it is determined that the underlying Airflow components are running the latest version of your code. During this time, you can hover over the status indicator to determine whether your entire Astro project or only DAGs were deployed .

> When the deploy completes, the **Docker Image** and **DAG Bundle Version** fields in the Cloud UI are updated with the Deployment code version. The **Docker Image** field displays a unique identifier generated by a Continuous Integration (CI) tool or a timestamp generated by the Astro CLI on `astro deploy`. The **DAG Bundle Version** field displays a unique timestamp generated by the Astro CLI when DAGs are updated. If you do not have DAG-only deploys enabled, the DAG bundle version field isn't populated.

Reviewer 2: ```suggestion
When the deploy completes, the **Docker Image** and **DAG Bundle Version** fields in the Cloud UI are updated depending on what type of deploy you completed. 

- The **Docker Image** field displays a unique identifier generated by a Continuous Integration (CI) tool or a timestamp generated by the Astro CLI after you complete an image deploy. 
- The **DAG Bundle Version** field displays a unique timestamp generated by the Astro CLI after you complete an image deploy or a DAG-only deploy. If you do not have DAG-only deploys enabled, the DAG bundle version field isn't populated.
```

> To confirm a deploy was successful, verify that the running versions of your Docker image and DAGs are the latest versions.

Reviewer 2: ```suggestion
To confirm a deploy was successful, verify that the running versions of your Docker image and DAG bundle have been updated.
```

1. In the Cloud UI, select a Workspace and then select a Deployment.

> 2. Review the information in the **Docker Image** field.

Author: ```suggestion
2. Review the information in the **Docker Image** field to determine the the Deployment code version. The unique identifier in this field is generated by a Continuous Integration (CI) tool or a timestamp generated by the Astro CLI on `astro deploy`.
```

> 2. Review the information in the **Docker Image** field to determine the the Deployment code version. The unique identifier in this field is generated by a Continuous Integration (CI) tool or a timestamp generated by the Astro CLI on `astro deploy`.

Reviewer 2: ```suggestion
2. Review the information in the **Docker Image** and **DAG bundle version** field to determine the Deployment code version.
```
